yarn.lock

```diff
@@ -1,12538 +0,0 @@
-# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Seems like you accidentally delete `yarn.lock`

Reply:
sorry. reverted